sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -311,6 +304,38 @@ public JournalPersonDto getPersonForJournal(String uuid) {
 			return null;
 		}
 	}
+	
+	private String getJournalEmailAddress(PersonDto person) {
+		try {
+			return person.getEmailAddress(false);
+		} catch (PersonDto.SeveralNonPrimaryContactDetailsException e) {
+			logger.error("Error retrieving email address", e);
+			return StringUtils.EMPTY;
+		}
+	}
+	
+	private String getJournalPhoneNumber(PersonDto person) {
+		try {
+			String phoneNumber = person.getPhone(false);
+			if (StringUtils.EMPTY.equals(phoneNumber)) {
+				return phoneNumber;
+			}
+
+			if (configFacade.getPatientDiaryConfig().isActive()) {
+				PhoneNumberUtil phoneUtil = PhoneNumberUtil.getInstance();
+				Phonenumber.PhoneNumber numberProto = phoneUtil.parse(phoneNumber, "DE");
+				return phoneUtil.format(numberProto, PhoneNumberUtil.PhoneNumberFormat.INTERNATIONAL);
+			} else {
+				return phoneNumber;
+			}
+		} catch (PersonDto.SeveralNonPrimaryContactDetailsException e) {
+			logger.error("Error retrieving phone number", e);
+			return StringUtils.EMPTY;
+		} catch (NumberParseException e) {
+			logger.error("Error while trying to parse phone number as INTERNATIONAL", e);
```

Comment:
Why did you change the behaviour from 
`exportPerson.setPhone(detailedPerson.getPhone());` to logging an error message here?